integration/common/openlineage/common/test.py

```diff
@@ -80,7 +80,7 @@ def setup_jinja() -> Environment:
 env = setup_jinja()
 
 
-def match(expected, result) -> bool:
+def match(expected, result, ordered_list=False) -> bool:
```

Comment:
I have use cases where I want to test list equality (order and every element in it)